faunadb-java-dsl/src/main/java/com/faunadb/client/util/SymbolGenerator.java

```diff
@@ -0,0 +1,15 @@
+package com.faunadb.client.util;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+final public class SymbolGenerator {
+    private static final AtomicLong counter = new AtomicLong(0);
```

Comment:
If you're not going for the reflection API as suggested above, do we need to ensure we don't go over the max long value? It's very unlikely but, technically possible?

Reply:
It just overflows, which still produces a valid symbol.